.eslintrc.js

```diff
@@ -25,25 +25,59 @@ module.exports = {
       },
     },
   },
-  extends: ['react-app', 'airbnb', 'prettier', 'prettier/react'],
```

Comment:
what's the story with 'prettier/react'?

Reply:
The disparate configs supplied by `eslint-config-prettier` were [merged](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21) in v8 of that config, so we no longer need to specify them separately.